README.md

```diff
@@ -1 +1,39 @@
-# gix-components
+# Gix Components
+
+Gix Components is a UI kit developed in Svelte by the Gix team.
+
+## Documentation
+
+The component library is in active development, and new features will incrementally be available.
+
+A full documentation will be available shortly.
+
+## Installation
+
+Install `gix-components` as a dependency.
+
+```bash
+npm i @dfinity/gix-components
+```
+
+## Usage
+
+### Styling
+
+The styles provided by the kit are not pre-compiled - i.e. is not provided as `CSS` files but as `SCSS`. Make sure that a SASS preprocessing is installed on your side.
```

Comment:
Does it mean that sass should be installed with `-g`?

Reply:
Oh no, just in the consuming project. Modified the sentence from "on your side" to "in your project". Thanks